Content.Server/StationRecords/Systems/StationRecordsSystem.cs

```diff
@@ -102,8 +106,9 @@ private void OnPlayerSpawn(PlayerSpawnCompleteEvent args)
     ///     about the player character.
     ///     Optional - other systems should anticipate this.
     /// </param>
+    /// <param name="status">Status of the person (None/Wanted/Detained).</param>
     /// <param name="records">Station records component.</param>
-    public void CreateGeneralRecord(EntityUid station, EntityUid? idUid, string name, int age, string species, Gender gender, string jobId, string? mobFingerprint, HumanoidCharacterProfile? profile = null,
+    public void CreateGeneralRecord(EntityUid station, EntityUid? idUid, string name, int age, string species, Gender gender, string jobId, SecurityStatus? status, string? mobFingerprint, HumanoidCharacterProfile? profile = null,
```

Comment:
no - what you need to do is create a separate system to interact with this related to security records

do not expand this any further